backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/StatusAndDiscoverIntegrationTestSuite.java

```diff
@@ -48,8 +48,8 @@
   private IGitMacheteRepositorySnapshot gitMacheteRepositorySnapshot;
 
   @BeforeClass
-  public static void ensureExpectedCliVersion() {
-    ensureCliVersionIs("2.14.0");
+  public static void doEnsureExpectedCliVersion() {
```

Comment:
Can't we move this method (and the one from `UpstreamInferenceIntegrationTestSuite.java`) higher, to `BaseGitRepositoryBackedIntegrationTestSuite`?

Reply:
Actually.. `UiTestSuite` also extends `BaseGitRepositoryBackedIntegrationTestSuite`, but as for now it doesn't use CLI at all (although it might need in future: #397)

Reply:
So I'd rather use it on a per-suite basis